conda-forge.yml

```
@@ -1,6 +1,12 @@
+build_platform:
+  linux_aarch64: linux_64
+  linux_ppc64le: linux_64
+conda_build:
+  pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
-conda_build:
-  pkg_format: '2'
+provider:
+  linux_aarch64: default
+  linux_ppc64le: default
```

Looks like it is trying to run tests when cross-compiling, which won't work as it is not on the right hardware

Think we need to limit testing to native and emulated cases [like what the macOS ARM PR does]( https://github.com/conda-forge/rabbitmq-server-feedstock/pull/112/files#diff-478786365dd93f740eb520c2faa03d7a0623273663b75472272c8ef94297bbe2R9 ). Have included this as a suggestion below

```suggestion
  linux_ppc64le: default
test: native_and_emulated
```

Note: This needs a re-render after being committed